docs/src/howto-guides/child_device_firmware_management.md

```diff
@@ -0,0 +1,199 @@
+# Enable firmware management on child devices with Cumulocity
+
+Firmware management can be enabled for child devices using the `c8y-firmware-plugin`.
+This thin-edge plugin coordinates the firmware update operation handling with Cumulocity,
+by establishing secure communication with the cloud,
+managing firmware file downloads, which are typically large files, even over flaky networks,
+caching the downloaded files for re-use across multiple child devices etc.
+
+But, an additional piece of software must be developed by the child device owner as well,
+to coordinate firmware management operations on it with the `c8y-firmware-plugin` running on thin-edge.
+This software is referred to as `child-device-connector` in the rest of this document.
+
+The responsibilities of the child device connector are:
+
+* Receive firmware update requests from thin-edge
+* Download and apply the updated firmware from thin-edge
+* Send status updates on the progress of the firmware update operation to thin-edge
+
+Handling the above mentioned responsibilities involve
+multiple interactions with thin-edge over its MQTT and HTTP APIs.
+In cases, where the child device connector is installed alongside thin-edge on the same device,
+these APIs can be accessed via a local IP or even `127.0.0.1`.
+The MQTT APIs are exposed via port 1883 and the HTTP APIs are exposed via port 8000.
```

Comment:
I think better to add a word "default" somehow to impress stronger that they are configurable.
```suggestion
The MQTT APIs are exposed via port 1883 and the HTTP APIs are exposed via port 8000 by default.
```